nixos/modules/hardware/video/nvidia.nix

```diff
@@ -70,6 +71,7 @@ in {
           NVSWITCH_FAILURE_MODE=0;
           ABORT_CUDA_JOBS_ON_FM_EXIT=1;
           TOPOLOGY_FILE_PATH=nvidia_x11.fabricmanager + "/share/nvidia-fabricmanager/nvidia/nvswitch";
+          DATABASE_PATH=nvidia_x11.fabricmanager + "/share/nvidia-fabricmanager/nvidia/nvswitch";
```

Comment:
Example should be updated to match actual configuration

Reply:
> Example should be updated to match actual configuration

I cannot compile then, I tried before.

Reply:
I think you just need to escape the formatting symbol like `''$`